README.md

```diff
@@ -74,6 +74,12 @@ var defaults = {
   //   - reposition: 'viewport'
   // - default/undefined => on a position event for any node (not as efficient...)
   when: undefined
+
+  // specify a set of nodes over which the rule may apply. This is useful for large graphs, 
+  // where only a small, known subset of nodes may be affected by a given automove-rule. 
+  // - a collection of nodes
+  // - default/undefined => all nodes in graph
+  scope: undefined
```

Comment:
This is exactly the same as the existing `nodesMatching` option, so I don't see the point in adding another one.  I think it would be better to just allow passing a collection as `nodesMatching` instead.